src/main/java/firebase/sdk/dashboard/data/Platform.java

```diff
@@ -1,9 +1,25 @@
 package firebase.sdk.dashboard.data;
 
 public enum Platform {
-  ANDROID,
-  IOS,
-  WEB,
-  GAMES
+  UNKNOWN(0),
+  ANDROID(1),
+  IOS(2),
+  WEB(3),
+  GAMES(4);
+
+  private int value;
+  private static final Platform[] platformValues = Platform.values();
```

Comment:
Why are you getting the list of platforms in the ENUM class?

Reply:
This was in order to create a mapping from integer to platform and vice versa

Reply:
Since the datastore could not store enums, opted for storing them as integers instead

Reply:
I think we finalized on Strings right. Just closing in on the discussion.